test/unit/org/apache/cassandra/index/sai/cql/DBPE14965Test.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright DataStax, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.cassandra.index.sai.cql;
+
+import org.junit.Test;
+
+import org.apache.cassandra.index.sai.SAITester;
+import org.assertj.core.api.Assertions;
+
+public class DBPE14965Test extends SAITester
```

Comment:
@adelapena @pkolaczk, this test does not fail upstream.

Upstream we have `StaticColumnIndexTest`, which was added as part of CASSANDRA-19278, which looks to me what addressed this.

I'm curious why wasn't that pulled in instead ?  And whether we should, and how we know, to remove this test in `main-5.0` ?